lib/sync/installation.js

```diff
@@ -195,35 +202,40 @@ const processInstallation = (app, queues) => {
           throw err;
         }
       }
+
       await updateJobStatus(jiraClient, job, edges, task, repositoryId);
     } catch (err) {
       const reteLimit = +(err.headers && err.headers['x-ratelimit-reset']);
       const delay = Math.max(Date.now() - reteLimit * 1000, 0);
+
       if (delay) { // if not NaN or 0
-        app.log(`Delaying job for ${delay}ms installationId=${installationId}, repositoryId=${repositoryId}, task=${task}`);
```

Comment:
What level would `app.log` log to? Wouldn't it be just `info`? But you translated all these calls to `logger.error()`.